### PR TITLE
Fix require() of readable-stream to allow usage of node-core stream

### PR DESCRIPTION
**Pull request by heri16**

### CODE REVIEW

File: bl.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var DuplexStream = require('readable-stream/duplex')
+var DuplexStream = require('stream').Duplex || require('readable-stream').Duplex
```
Comment (mcollina) [inline bl.js]: Unfortunately no. It should be `require('readable-stream').Duplex` if you want to change this.
This is not really a fallback, but a dependency in the true sense.

**Comment by heri16** [comment]
Is there a reason for pinning to userland? Were there changes to the Streams API since the oldest nodejs LTS version till current?

**Comment by heri16** [comment]
I have revised the code to get Duplex from main readable.js .

https://github.com/nodejs/readable-stream/blob/b3cf9b1f46eaa1865930ae03b96d7a4a570746f0/readable.js#L7

This allows the application code to give precedence to core-land stream rather than userland stream.

**Comment by mcollina** [comment]
There is no need for this. Duplex will always be present in readable-stream.
The reason why this is more stable is that bl inherits from Duplex. If we depended from the one in core, this module would need maintainance/work because of any semver-major changes that land there. By depending on readable-stream, this module became very stable.

**Comment by heri16** [comment]
Agreed. Enough to just have var DuplexStream = require('readable-stream').Duplex

Instead of the current var DuplexStream = require('readable-stream/duplex')

File: bl.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-var DuplexStream = require('readable-stream/duplex')
+var DuplexStream = require('readable-stream').Duplex || require('stream').Duplex
```
Comment (mcollina) [inline bl.js]: Can you please remove `require('stream').Duplex`?

**Comment by heri16** [comment]
Yes. Will do so once I get to my laptop

**Comment by heri16** [comment]
Updated with much thanks.

**Pull request merged**